modules/swagger-codegen/src/main/resources/Java/pojo.mustache

```diff
@@ -118,9 +118,16 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#parcela
 {{#vendorExtensions.extraAnnotation}}
   {{{vendorExtensions.extraAnnotation}}}
 {{/vendorExtensions.extraAnnotation}}
+  {{#isBoolean}}
```

Comment:
@wing328 I don't understand where you could configure to use either `getMyBooleanProperty()` or `isMyBooleanProperty()`.
Am I missing something?

Reply:
You can use the -t option with customized templates as follows to have `getMyBooleanProperty`:
```
  {{#isBoolean}}
  public {{{datatypeWithEnum}}} get{{getter}}() {
    return {{name}};
  }
  {{/isBoolean}}
  {{^isBoolean}}
  public {{{datatypeWithEnum}}} {{getter}}() {
    return {{name}};
  }
  {{/isBoolean}}
```

Reply:
don't you think it would be easier to have a configuration option to choose between the two possibilities?
as a developer I wouldn't want to have to copy and customize a swagger mustache template to simply configure a standard-variant. Customizing requires additional efforts when e.g. upgrading. 
Moreover, the using `is...()` with a non-primitive `Boolean` instead of a primitive `boolean` is not specified and can thus lead to discussions concerning support in e.g. data mappers like here: https://jira.spring.io/browse/SPR-9482

Reply:
@cbornet what is your opinion on that?

Reply:
> don't you think it would be easier to have a configuration option to choose between the two possibilities?

Agreed that adding an option is one way but personally I try to avoid too many options as there're already quite a lot of options for Java client generator already.

Besides `is`, `get`, another possible choice is `has`.

Reply:
@wing328 trying to keep the number of options as small as possible is a good approach. however, as the option in this case would allow using standard-behavior of data mappers, (which used to work before), I consider it worth adding instead of requiring customizations.
Personally, I think it is one of the strengths of swagger codegen to be able to generate standard-code from a swagger-file and directly use it without customizations.
And using the default data mapper of Spring Boot definitely is such a standard-setup.